bokeh_garden/text_input.py

```diff
@@ -0,0 +1,13 @@
+import bokeh.models
```

Comment:
Oups, did you really mean to commit this file? I think you should remove this one before we merge this PR...